### PR TITLE
fixing api paths to match the new public route pattern, as well as mi…

### PR DESCRIPTION
**Pull request by Codiegladney-bluescape**
…scellaneous other fixes to grammar, hyperlinks, etc.

### CODE REVIEW

File: site-templates/pages/what-is-new-in-v3.gomd.debughtml [hunk 1]
```diff
@@ -20,9 +20,12 @@ The main differences can be grouped in 2 sections:
     <td><center><b>V2</b></center></td>
     <td><center><b>V3</b></center></td>
   <tr>
-    <td WIDRTH="30%">In V3, the REST API verb to modify an element already created (its position, color, text, etc.) is PATCH.</td>
-    <td>verb: PUT</td>
-    <td>verb: PATCH</td>
+    <td WIDRTH="30%">In V3, when you want to modify a property of an object that is <br />already created (its position, color, text, etc.), the REST API verb to use is PATCH.</td>
```
Comment (Jackychen-bluescape) [inline site-templates/pages/what-is-new-in-v3.gomd.debughtml:23]: "WIDRTH" typo?

Comment (Codiegladney-bluescape) [reply site-templates/pages/what-is-new-in-v3.gomd.debughtml:23]: Must be typo in the actual file itself

Comment (Codiegladney-bluescape) [reply site-templates/pages/what-is-new-in-v3.gomd.debughtml:23]: Can ignore the debughtml files. I need to make sure they're being ignored when committing and requesting review

Comment (NoahBluescape) [reply site-templates/pages/what-is-new-in-v3.gomd.debughtml:23]: I just added them to .gitignore in my last PR. Should help hopefully

Comment (Jackychen-bluescape) [reply site-templates/pages/what-is-new-in-v3.gomd.debughtml:23]: Yeah, think @NoahBluescape added .debughtml in gitinore for the last PR

**Review by Jackychen-bluescape — APPROVED** [review]
LGTM.

**Pull request merged**